### PR TITLE
arangodb 3.10.2

### PR DESCRIPTION
**Pull request by ryenus**
- [ ] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [ ] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [ ] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [ ] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [ ] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----
Migrate arangodb to python@3.11
See https://github.com/Homebrew/homebrew-core/pull/114154

### CODE REVIEW

**Comment by chenrui333** [comment]
linux build failed.

**Comment by iMichka** [comment]
Same error as in #118760

**Comment by iMichka** [comment]
Trying to build with gcc 10.

File: Formula/arangodb.rb [hunk 1]
```diff
@@ -41,6 +45,13 @@ class Arangodb < Formula
     EOS
   end
 
+  # https://github.com/arangodb/arangodb/issues/17454
+  # https://github.com/arangodb/arangodb/issues/17454
+  fails_with :gcc do
+    version "11"
```
Comment (cho-m) [inline Formula/arangodb.rb:51]: This will reject `gcc <= 11`. With current DSL, will want to use `fails_with gcc: "11"` which rejects `gcc == 11`.

**Review by cho-m — APPROVED** [review]

**Comment by cho-m** [comment]
11-arm64 only failed on CI cleanup and it looks like bottles uploaded, so trying a merge.

**Pull request closed without merging**